ssg/constants.py

```diff
@@ -60,19 +60,21 @@
 ]
 
 JINJA_MACROS_BASE_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros.jinja")
+    __file__)), "shared", "macros", "macros.jinja")
 JINJA_MACROS_HIGHLEVEL_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros-highlevel.jinja")
+    __file__)), "shared", "macros", "macros-highlevel.jinja")
 JINJA_MACROS_ANSIBLE_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros-ansible.jinja")
+    __file__)), "shared", "macros", "macros-ansible.jinja")
 JINJA_MACROS_IGNITION_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros-ignition.jinja")
+    __file__)), "shared", "macros", "macros-ignition.jinja")
 JINJA_MACROS_KUBERNETES_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros-kubernetes.jinja")
+    __file__)), "shared", "macros", "macros-kubernetes.jinja")
 JINJA_MACROS_OVAL_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros-oval.jinja")
+    __file__)), "shared", "macros", "macros-oval.jinja")
 JINJA_MACROS_BASH_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
-    __file__)), "shared", "macros-bash.jinja")
+    __file__)), "shared", "macros", "macros-bash.jinja")
+JINJA_MACROS_FIXTEXT_DEFINITIONS = os.path.join(os.path.dirname(os.path.dirname(
+    __file__)), "shared", "macros", "macros-fixtext.jinja")
```

Comment:
I wonder if we need to specify every jinja file we load.
We could simply load every `*.jinja` file from `shared/macros`, thoughts?

Reply:
that's  a great idea

Reply:
Done in 2827f07f9c10126a90b9d5338b7ee2dbb3d0b8b5